content/docs/hooks-reference.md

```diff
@@ -95,45 +96,45 @@ const [state, setState] = useState(() => {
 useEffect(didUpdate);
 ```
 
-Accepts a function that contains imperative, possibly effectful code.
+Acepta una función que contiene código imperativo, posiblemente código efectivo.
 
-Mutations, subscriptions, timers, logging, and other side effects are not allowed inside the main body of a function component (referred to as React's _render phase_). Doing so will lead to confusing bugs and inconsistencies in the UI.
+Las mutaciones, suscripciones, temporizadores, registro y otros efectos secundarios no están permitidos dentro del cuerpo principal de un componente de función (denominado como _render phase_ de React). Si lo hace, dará lugar a errores confusos e inconsistencias en la interfaz de usuario.
 
-Instead, use `useEffect`. The function passed to `useEffect` will run after the render is committed to the screen. Think of effects as an escape hatch from React's purely functional world into the imperative world.
+En su lugar, use `useEffect`. La función es pasada a `useEffect` se ejecutará después de que el renderizado se confirma en la pantalla. Piense en los efectos como una escotilla de escape del mundo puramente funcional de React al mundo imperativo.
```

Comment:
For me 
> La función pasada a `useEffect`...

is closer to the original than 

> La función es pasada a `useEffect` 

What do you think?

`el renderizado se confirme` or `el renderizado es confirmado` sounds better in my opinion